resources/sass/pages/_theme-detail.scss

```diff
@@ -0,0 +1,219 @@
+
+        .ris-link {
+            width: 100%;
+        }
+
+        .ris-img {
+            width: 100%;
+        }
+    }
+
+    .ris-title {
+        margin-bottom: $indent-s;
+    }
+
+    .ris-caption {
+        margin-top: $indent-s;
+
+        &:not(:last-child) {
+            margin-bottom: $indent-s;
+        }
+    }
+
+    .ris-map-detail {
+        @include breakpoint($bp-tablet-min) {
+            display: flex;
+            justify-content: center;
+        }
+
+        .ris-img {
+            border-radius: $b-radius-primary;
+        }
+
+        .ris-link {
+            @include breakpoint($bp-tablet-min) {
+                width: 300px;
+            }
+        }
+    }
+
+    .ris-map-all-street-address {
+        display: flex;
+
+        @include breakpoint($bp-mobile-max) {
+            margin-bottom: $indent-m;
+        }
+
+        &::before {
+            color: $c-secondary;
+            content: 'a';
+            display: inline-block;
+            font-family: $ff-ris-font-icons;
+            font-size: $fs-xl;
+            margin-right: $indent-sm;
+        }
+    }
+
+    .ris-action-box {
+        margin-bottom: $indent-m;
+
+        @include breakpoint($bp-mobile-max) {
+            flex-direction: column;
+        }
+
+        @include breakpoint($bp-tablet-min) {
+            align-items: center;
+            justify-content: space-between;
+        }
+
+        .ris-title {
+            margin-bottom: 0;
+
+            @include breakpoint($bp-mobile-max) {
+                margin-bottom: $indent-m;
+            }
+        }
+    }
+
+    .ris-document {
+        align-items: center;
+        display: flex;
+        justify-content: space-between;
+        margin-bottom: $indent-sm;
+        margin-top: $indent-sm;
+
+        &-many,
+        &-one {
+            .ris-icon-wrapper {
+                align-items: center;
+                display: flex;
+                height: 48px;
+                justify-content: center;
+                margin-right: $indent-m;
+                min-width: 48px;
+                width: 48px;
+            }
+        }
+
+        &::after {
+            color: $c-secondary;
+            content: 'f';
+            display: inline-block;
+            font-family: $ff-ris-font-icons;
+            font-size: $fs-xl;
+            margin-left: $indent-sm;
+        }
+
+        .ris-title {
+            font-size: $fs-l;
+            line-height: $lh-m;
+            margin-bottom: 0;
+        }
+
+        .ris-text-wrapper {
+            flex-grow: 1;
+        }
+    }
+
+    .ris-process {
+        &__meeting-summary {
+            display: flex;
+
+            @include breakpoint($bp-mobile-max) {
+                flex-direction: column;
+            }
+
+            @include breakpoint($bp-tablet-min) {
+                justify-content: center;
+            }
+
+            .ris-link {
+                @include breakpoint($bp-tablet-min) {
+                    width: 300px;
```

Comment:
300px also used in other places of this file, if they somehow related please make a variable for it